charts/fluid/fluid/values.yaml

```diff
@@ -152,12 +152,6 @@ webhook:
       # fluid existed node affinity, the name can not be modified.
       - name: fluid.io/node
         weight: 100
-      # runtime worker's rack label name, can be changed according to k8s environment.
-      - name: topology.kubernetes.io/rack
-        weight: 50
-      # runtime worker's zone label name, can be changed according to k8s environment.
-      - name: topology.kubernetes.io/zone
```

Comment:
Can you change the default configuration as

```yaml
# runtime worker's rack label name, can be changed according to k8s environment.
      - name: topology.kubernetes.io/zone
        weight: 50
      # runtime worker's zone label name, can be changed according to k8s environment.
      - name: topology.kubernetes.io/region
         weight: 20
     ```

Reply:
fixed.